packages/common/src/logger.ts

```diff
@@ -13,3 +13,44 @@ export interface Logger {
   warn(message: string, meta?: LogMetadata): any;
   error(message: string, meta?: LogMetadata): any;
 }
+
+/**
+ * Possible values of the `logSource` meta attributes on log messages. This
+ * attribute indicates which subsystem emitted the log message; this may for
+ * example be used to implement fine-grained filtering of log messages.
+ *
+ * Note that there is no guarantee that this list will remain stable in the
+ * future; values may be added or removed, and messages that are currently
+ * emitted with some `logSource` value may use a different value in the future.
+ */
+export enum LogSource {
+  /**
+   * Log Source value for messages emited from Workflow code, using the {@link Workflow
+   * context logger|workflow.log}. The SDK itself never publishes messages with this source.
```

Comment:
Can you verify this link is rendered properly?

Reply:
Fixed.